recipe/meta.yaml

```
@@ -19,14 +17,13 @@ requirements:
   build:
     - cmake
     - {{ compiler('c') }}
-    - ninja-base
-    - patch  # [linux64 or (linux and aarch64)]
+    - ninja
```

```suggestion
    - ninja-base
```

ninja-base goes without Python.